### PR TITLE
add kane_census job and license

### PR DESCRIPTION
**Pull request by saylorsd**
…. updated readme to account for new config format

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2016 Ben Smithgall, Steve Saylor
```
Comment (bsmithgall) [inline LICENSE.txt:1]: :+1: Though I think technically you would come first alphabetically @saylorsd

File: test/jobs/test_kane_census_pipeline.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                [('date', '2015-10-01'), ('gh_male', 81), ('gh_female', 109), ('gh_white', 116), ('gh_black', None),
+                 ('gh_other', 1), ('scott_male', 93), ('scott_female', 180), ('scott_white', 262), ('scott_black', 10),
+                 ('scott_other', 1), ('mck_male', 100), ('mck_female', 184), ('mck_white', 222), ('mck_black', 60),
+                 ('mck_other', 2), ('ross_male', 63), ('ross_female', 166), ('ross_white', 213), ('ross_black', 14),
+                 ('ross_other', 2)]
+            )
+        )
+        self.assertEquals(len(status), 1)
+        self.assertEquals(status[0][-2], 'success')
+        self.assertEquals(status[0][-1], 23)
```
Comment (bsmithgall) [inline test/jobs/test_kane_census_pipeline.py:37]: :+1:

**Comment by bsmithgall** [comment]
:+1: Looks good to me. 

This also fixes #22

**Comment by saylorsd** [comment]
@bsmithgall What timing! Thanks for taking a look. However, I just pushed a small change where `extractor.create_schema_headers()` removes trailing whitespace as it ended up being an issue with some of the source files we're receiving.

File: LICENSE.txt (hunk 1 above)
Comment (saylorsd) [inline LICENSE.txt:1]: I figured since you did the bulk of it, I'd put you first.

Comment (bsmithgall) [inline LICENSE.txt:1]: :heart: 

Don't discount your own efforts though!

**Comment by bsmithgall** [comment]
New patch looks good as well, though you might want to add some unit tests to prevent future regressions.

**Comment by saylorsd** [comment]
As in to ensure that it make the proper formats, strippings and replacements?

**Comment by bsmithgall** [comment]
Yup.

**Pull request merged**